src/js/components/FragmentList/fragmentList.style.jsx

```diff
@@ -0,0 +1,45 @@
+import styled from 'styled-components';
+
+const FragmentList = styled.div`
```

Comment:
It seems like there is different formatting for different element, two spaces or four spaces.